src/main.rs

```diff
@@ -490,13 +498,23 @@ async fn get_eth_head(State(state): State<Arc<AppState>>) -> impl IntoResponse {
                             if err.to_string().ends_with("status code: 429") {
                                 (
                                     StatusCode::TOO_MANY_REQUESTS,
-                                    [("Cache-Control", "max-age=300, must-revalidate")],
+                                    [
+                                        (
+                                            "Cache-Control",
+                                            "max-age=300, must-revalidate".to_string(),
```

Comment:
Doesn't work without `to_string()`?

Reply:
no, because of `format!` that returns `String` and when converting to `to_str` needs a different lifetime